build/js/validation.js

```diff
@@ -0,0 +1,149 @@
+'use strict';
+
+function validateNames(firstName, lastName) {
+    // checks if name fields are not empty and dont contain digits
+    var containsNumbers = function containsNumbers(nameToTest) {
+        return (/[0-9]/.test(nameToTest)
```

Comment:
Here containsNumbers() will return a RegEx.test() result which is not a boolean. It will work anyway because javascript handles null as "falsy" and a positive regex result as "truthy", but it would be cleaner to cast it to boolean :

```javascript
return !!(/[0-9]/.test(nameToTest));
```